src/common/config.h

```diff
@@ -46,6 +49,8 @@ void setMainWindowHeight(u32 height);
 void setPkgViewer(std::vector<std::string> pkgList);
 void setElfViewer(std::vector<std::string> elfList);
 void setRecentFiles(std::vector<std::string> recentFiles);
+void setKeyboardBindingMap(std::map<Uint32, KeysMapping> map);
+std::map<Uint32, KeysMapping> getKeyboardBindingMap();
```

Comment:
You are returning/giving a full std::map copy here which is super expensive
```suggestion
void setKeyboardBindingMap(std::map<u32, KeysMapping>&& map);
const std::map<u32, KeysMapping>& getKeyboardBindingMap();
```

Reply:
you're right, const ref is better. Thank you for your remark!